packages/picasso/src/utils/index.ts

```diff
@@ -66,3 +66,27 @@ export {
   useDeprecationWarning,
   usePropDeprecationWarning,
 } from './use-deprecation-warnings'
+
+const {
+  SPACING_0,
+  SPACING_1,
+  SPACING_2,
+  SPACING_3,
+  SPACING_4,
+  SPACING_5,
+  SPACING_6,
+  SPACING_7,
+  SPACING_8,
+} = spacings
+
+export {
+  SPACING_0,
+  SPACING_1,
+  SPACING_2,
+  SPACING_3,
+  SPACING_4,
+  SPACING_5,
+  SPACING_6,
+  SPACING_7,
+  SPACING_8,
+}
```

Comment:
You could simplify it to:
```suggestion
export const {
  SPACING_0,
  SPACING_1,
  SPACING_2,
  SPACING_3,
  SPACING_4,
  SPACING_5,
  SPACING_6,
  SPACING_7,
  SPACING_8,
} = spacings
```